### ISSUE TITLE
Make error messages more useful

### ISSUE THREAD

**Issue by mbch331** [OpenRefine/OpenRefine#5166]
When an error occurs it's not clear which edit gives a problem, I'm trying to correct 73 records and everytime at 40% it gets stuck. In the console I get an error, but no idea which edit exactly gives the problem.

### Proposed solution
Add the statement that is being sent to Wikidata in the error message, so it's easier to debug where the error could be.

### Alternatives considered
Add the Qid of the statement that gives the problem.

### Additional context
```
Exception in thread "Thread-8" java.lang.IllegalArgumentException: Statements describe different subject.
        at org.apache.commons.lang3.Validate.isTrue(Validate.java:158)
        at org.wikidata.wdtk.datamodel.implementation.StatementDocumentUpdateImpl.<init>(StatementDocumentUpdateImpl.java:68)
        at org.wikidata.wdtk.datamodel.implementation.LabeledDocumentUpdateImpl.<init>(LabeledDocumentUpdateImpl.java:64)
        at org.wikidata.wdtk.datamodel.implementation.TermedDocumentUpdateImpl.<init>(TermedDocumentUpdateImpl.java:78)
        at org.wikidata.wdtk.datamodel.implementation.ItemUpdateImpl.<init>(ItemUpdateImpl.java:91)
        at org.wikidata.wdtk.datamodel.implementation.DataObjectFactoryImpl.getItemUpdate(DataObjectFactoryImpl.java:405)
        at org.wikidata.wdtk.datamodel.helpers.Datamodel.makeItemUpdate(Datamodel.java:1219)
        at org.openrefine.wikidata.updates.ItemEdit.toEntityUpdate(ItemEdit.java:165)
        at org.openrefine.wikidata.editing.EditBatchProcessor.performEdit(EditBatchProcessor.java:173)
        at org.openrefine.wikidata.operations.PerformWikibaseEditsOperation$PerformEditsProcess.run(PerformWikibaseEditsOperation.java:262)
        at java.base/java.lang.Thread.run(Unknown Source)
```

**Comment by wetneb** [comment]
This particular bug should normally be closed by #5163 but I totally agree with you that we really need to up our error-handling game, in a ton of places.

**Comment by raymondde** [comment]
Same issue with uploading to Wikimedia Commons now.  During the upload runs I have seen different errors in the console:

1. http-bad-status: There was a problem during the HTTP request: 415 Unsupported Media Type
2. verification error: File extension ".jpg" does not match the detected MIME type of the file (inode/x-empty)
3. abusefilter-warning: abusefilter-warning-overwriting-artwork

Screenshots: https://photos.app.goo.gl/84rzjVweQUrUFC338

The reasons for 1. and 2. are currently not checked. Probably corrupt files. The reason for 3. is known. My fault because I forgot to exclude some already uploaded files.
What I am missing: a saved error log which includes the file names. Currently I have no idea which files are generating the errors from 1. and 2.

**Comment by hrabian** [comment]
> This particular bug should normally be closed by #5163 but I totally agree with you that we really need to up our error-handling game, in a ton of places.

This bug is still present in 3.6.1 (5fc8883)
